std/zip.d

```diff
@@ -672,14 +718,8 @@ public:
             _numEntries = getUshort(i + 8);
             _totalEntries = getUshort(i + 10);
 
-            if (numEntries != totalEntries)
```

Comment:
Why is this check (and multiple other checks thorughout the file) removed?

Reply:
That are checks, that can never happen, because they are allready filtered by findEndOfCentralDirRecord or by the calls to removeSegment.

Reply:
Okay